gpiozero/output_devices.py

```diff
@@ -503,6 +503,243 @@ def _blink_device(
             self._write(value)
             if self._blink_thread.stopping.wait(delay):
                 break
+            
+class PWMBuzzer(PWMOutputDevice):
+    """
+    Extends :class: `PWMOutputDevice` and represents a piezo or tonal buzzer 
+    class.
```

Comment:
Isn't this only for tonal buzzers? AIUI piezo buzzers constantly beep whenever a fixed voltage is applied to them?